app_dart/test/request_handlers/update_task_status_test.dart

```diff
@@ -4,34 +4,42 @@
 
 import 'package:cocoon_service/src/model/appengine/commit.dart';
 import 'package:cocoon_service/src/model/appengine/task.dart';
+import 'package:cocoon_service/src/model/firestore/task.dart' as f;
```

Comment:
Same comment as above for the import name.

Reply:
Done